src/status_im/translations/fr.cljs

```diff
@@ -256,59 +236,133 @@
+   :unsigned-transaction-expired          "La transaction non signée a expirée"
    :status                                "Statut"
-   :pending-confirmation                  "Confirmation en attente"
    :recipient                             "Destinataire"
-   :one-more-item                         "Encore un élément"
-   :fee                                   "Frais"
-   :estimated-fee                         "Frais Estimés"
-   :value                                 "Valeur"
    :to                                    "à"
    :from                                  "de"
    :data                                  "Données"
    :got-it                                "Compris"
-   :contract-creation                     "Création du contrat"
+   :block                                 "Bloc"
+   :hash                                  "Hash"
+   :gas-limit                             "limite Gas"
+   :gas-price                             "prix Gas"
+   :gas-used                              "Gas utilisé"
+   :cost-fee                              "Coût/Frais"
+   :nonce                                 "Nonce"
+   :confirmations                         "Confirmations"
+   :confirmations-helper-text             "Veuillez attendre au moins 12 confirmations pour être sûr que votre transaction a bien été définitivement validée"
+   :copy-transaction-hash                 "Copier le hash de la transaction"
+   :open-on-etherscan                     "Ouvrir sur Etherscan.io"
 
    ;:webview
    :web-view-error                        "Oops, erreur"})
+   
+   ;;testfairy warning
+   :testfairy-title                       "Avertissement!"
+   :testfairy-message                     "Vous utilisez une application installée à partir d'une version de développement. À des fins de test, cette version inclut l'enregistrement de session si la connexion Wi-Fi est utilisée, ainsi toutes vos interactions avec cette application sont sauvegardées (vidéo et logs) et pourraient être utilisées par notre équipe de développement pour enquêter sur d'éventuels problèmes. Les vidéos / logs enregistrés n'incluent pas vos mots de passe. L'enregistrement est effectué uniquement si l'application est installée à partir d'une version nocturne. Rien n'est enregistré si l'application est installée depuis PlayStore ou TestFlight."
+
+   ;; wallet
+   :wallet                                "Portefeuille"
+   :wallets                               "Portefeuilles"
+   :your-wallets                          "Vos portefeuilles"
+   :main-wallet                           "Portefeuille principale"
+   :wallet-error                          "Erreur survenue lors du chargement de données"
+   :wallet-send                           "Envoyé"
+   :wallet-request                        "Requête"
+   :wallet-exchange                       "Echange"
+   :wallet-assets                         "Assets"
+   :wallet-add-asset                      "Ajouter un avoir"
```

Comment:
Above asset is not translated, maybe "actif" fits better than "avoir" for asset

Reply:
@jeluard do you agree?